### PR TITLE
[BM-597] Fix warnings in the project

### PR DESCRIPTION
**Pull request by anyashka**
### Ticket

https://netguru.atlassian.net/browse/BM-597
 
### Task Description

 Updated the swift lint file with rules and fixed warnings in the project.

### CODE REVIEW

File: Baby Monitor/Source Files/Custom Modules/Views/BabyNavigationItemView.swift [hunk 1]
```diff
@@ -88,7 +88,7 @@ final class BabyNavigationItemView: UIView {
         }
         switch mode {
         case .baby:
-            [pulsatoryView].forEach { // TODO: add nameTextField here if you want to show it
+            [pulsatoryView].forEach {
                 stackView.addArrangedSubview($0)
             }
```
Comment (3Qax) [inline Baby Monitor/Source Files/Custom Modules/Views/BabyNavigationItemView.swift]:
```suggestion
            stackView.addArrangedSubview(pulsatoryView)
```

File: Baby Monitor/Source Files/Custom Modules/View Controllers/TypedPageViewController.swift [hunk 2]
```diff
@@ -19,7 +19,7 @@ internal class TypedPageViewController<BackgroundView: UIView>: BasePageViewCont
     ///     - analyticsScreenType: The type of the screen in terms of analytics tracking.
     init(viewMaker: @escaping @autoclosure () -> BackgroundView,
          analytics: AnalyticsManager,
-         analyticsScreenType: AnalyticsScreenType){
+         analyticsScreenType: AnalyticsScreenType) {
```
Comment (3Qax) [inline Baby Monitor/Source Files/Custom Modules/View Controllers/TypedPageViewController.swift]: Are you sure that your version of formatting doesn't trigger warning? From what I remember it should.
```suggestion
         analyticsScreenType: AnalyticsScreenType
    ) {
```

Comment (anyashka) [reply Baby Monitor/Source Files/Custom Modules/View Controllers/TypedPageViewController.swift]: It doesn't :) Do you remember which rule triggers it?

Comment (3Qax) [reply Baby Monitor/Source Files/Custom Modules/View Controllers/TypedPageViewController.swift]: `multiline_parameters_brackets `

Comment (anyashka) [reply Baby Monitor/Source Files/Custom Modules/View Controllers/TypedPageViewController.swift]: This rule is in the list of disabled :)

**Pull request merged**